.github/workflows/cabal.project.local.ci.Darwin

```diff
@@ -3,3 +3,13 @@ package cardano-crypto-praos
 
 package HsOpenSSL
   flags: +use-pkg-config
+
+allow-newer:
```

Comment:
What we've been doing in some other places is to use the new cabal support for conditionals in project files to make it clear that these are only needed for 9.6+

Like this: https://github.com/haskell/haskell-language-server/blob/master/cabal.project#L118

Then I think you can sensibly put this in the main `cabal.project` rather than all these CI-specific ones.

Reply:
Yes, that is better